eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -429,8 +429,14 @@ multiasset<a> = { * policy_id => { * asset_name => a } }
 policy_id = scripthash
 asset_name = bytes .size (0..32)
 
-value = coin / [coin,multiasset<uint>]
-mint = multiasset<int64>
+negInt64 = -9223372036854775808 .. -1
+posInt64 = 1 .. 9223372036854775808
```

Comment:
I made a typo in my recommended changes to the CDDL in https://github.com/input-output-hk/cardano-ledger/issues/3240. This would cause an overflow, as the maximum value of a 64-bit integer is `9223372036854775807` 

The corrected definition is:
> posInt64 = 1 .. 9223372036854775807

Reply:
which maybe would have lead us to continue to allow zero :laughing: 

thank you for the heads up!

Reply:
Nah it would have just wrapped around into a `-9223372036854775808`. So instead of minting it would be trying to burn :smile: